prestodb/iop4.2-hive/README.md

```diff
@@ -13,7 +13,7 @@ for general correctness testing.
 ## Run
 
 ```
-$ docker run -d --name hadoop-master -h hadoop-master teradatalabs/iop-hive
+$ docker run -d --name hadoop-master -h hadoop-master prestodb/iop4.2-hive
```

Comment:
are there more `teradatalabs` occurences?

Reply:
found a couple, fixed in a separate commit